File: VampireUi/src/app/_models/Blood.ts
export class Blood {

	donorName: String;
	bloodtype: String;
	status: String;
	dateDonated: string;
    location: String;

    constructor(name: String, status: String, type: String,
    	dateDonatedOn: string, donatedAt: String) {
    
    	this.donorName = name;
    	this.bloodtype = type;
    	this.status = status;
    	this.dateDonated = dateDonatedOn;
    	this.location = donatedAt;

    }
}

File: VampireBackEnd/VampireBackEnd/Dtos/Blood.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace VampireBackEnd.Dtos
{
    public class Blood
    {
        public Guid bloodId { get; set; }
        public string donorName { get; set; }
        public string bloodType { get; set; }
        public string bloodStatus { get; set; }
        public string dateDonated { get; set; }
        public string locationAcquired { get; set; }
    }
}
